packages/client/src/pages/Safe.js

```diff
@@ -160,6 +160,36 @@ function Safe({ web3 }) {
     ).catch((error) => showErrorModal(error));
   };
 
+  const onRejectAction = async ({ uuid, intent }) => {
+    const latestBlock = await injectedProvider
```

Comment:
Any reason not to use the `createSignature` defined function in web3?

Reply:
There is a slight difference, because the message in approve/reject contains the uuid of the action, I'll try to consolidate those two into one function